install/downloads/toolbox-latex-install.sh

```diff
@@ -1,9 +1,11 @@
 #!/bin/bash
 
+set -euo pipefail
+
 
 # download installer
 
-cd ~/.local
+cd ~/tmp
```

Comment:
No! please create a new tmp directory and use that, not just the `/tmp`. Use `tmpdir = $(mktemp -d)`